scripts/sync_translations.py

```diff
@@ -0,0 +1,217 @@
+import configparser
+import os
+import sys
+from os.path import expanduser
+import yaml
+
+from transifex.api import transifex_api
+from transifex.api.jsonapi import exceptions
+
+NEW_PROJECT_SLUG = 'openedx-translations'
+ORGANIZATION_SLUG = 'open-edx'
+
+
+class Command:
```

Comment:
This command has not tests. I realize that testing it isn't straight forward, so I wanted to ask what ideas do you have to prepare some unit tests for this script?

Reply:
I think it will live longer that we'd like to. I'm fine with merging without tests if we end up removing the `cron` section.

This way, only admins (myself, you and @brian-smith-tcril) will run this workflow and they should be able to fix bugs if any.

Reply:
I disabled cron section

Reply:
I think we should at least add a basic test or two, even if we don't get full coverage in this PR. That way, once we start manually running this, if we encounter any bugs we'll already have a place to add a test that fails due to the discovered bug (which will make it easier to verify the bugfix).